common/app/model/meta.scala

```diff
@@ -130,6 +130,8 @@ final case class Fields(
 
 object MetaData {
 
+  val StartDateForHttpsFacebookUrls: OffsetDateTime = OffsetDateTime.of(2021, 9, 3, 12, 0, 0, 0, ZoneOffset.UTC)
```

Comment:
Using java.time rather than injecting a new Joda usage while we're trying to migrate away.